docker/python3-ubi/Pipfile

```diff
@@ -0,0 +1,21 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+requests = {extras = ["socks"],version = "*"}
+olefile = "*"
+python-dateutil = "*"
+dateparser = "*"
+pyyaml = "*"
+funcy = "*"
+tldextract = "*"
+pip = "*"
```

Comment:
Why is this file different from the Pipfile used by python3? 

See: https://github.com/demisto/dockerfiles/blob/master/docker/python3/Pipfile

Reply:
done